### PR TITLE
Blogpost for PyDelhi's Meetup held on June 9, 2018.

### PR DESCRIPTION
**Pull request by im-gozmit**
Please review.

### CODE REVIEW

**Review by realslimshanky — COMMENTED** [review]
Please squash your commits. Rest LGTM.

File: content/2018-06-09-pydelhi-meetup.md
Comment (dotgourav) [inline content/2018-06-09-pydelhi-meetup.md]: each person presents/gave(choose one)

File: content/2018-06-09-pydelhi-meetup.md
Comment (dotgourav) [inline content/2018-06-09-pydelhi-meetup.md]: Link PyCon India website here.

**Review by dotgourav — CHANGES_REQUESTED** [review]

**Comment by im-gozmit** [comment]
Thanks, @realslimshanky and @Gouravchawla for the edits.

**Comment by rajataaron** [comment]
@realslimshanky can you please check the PR once and merge it.

**Comment by realslimshanky** [comment]
@rajataaron I don't have merge access to this repo. @akshayaurora can you please help out here.

**Comment by akshayaurora** [comment]
PR is not in mergeable state, please fix conflicts.

File: content/2018-06-09-pydelhi-meetup.md
Comment (akshayaurora) [inline content/2018-06-09-pydelhi-meetup.md:11]: conflicts ?

**Comment by realslimshanky** [comment]
@im-gozmit please address reviews.

**Comment by aaqaishtyaq** [comment]
Hey @im-gozmit, can you address requested changes.

**Pull request merged**
